ayon_server/graphql/resolvers/tasks.py

```diff
@@ -71,7 +71,7 @@ async def get_tasks(
         return TasksConnection(edges=[])
 
     project_name = root.project_name
-    fields = FieldInfo(info, ["tasks.edges.node", "subset"])
+    fields = FieldInfo(info, ["tasks.edges.node", "task"])
```

Comment:
This was a bug before, or is bug now? (`subset` -> `task`)

Reply:
It was a bug before. This is used to check, which fields are requested in the query. The bug occurred rarely in the UI, so it was there for a long time - basically it caused folder attributes weren't propagated to task if the task was requested directly using its id.